### DISCUSSION TITLE
E3V2-BLTouch-5x5-v4.2.2 in 220x220x250

### DISCUSSION THREAD

**Discussion by ManuR93** [Jyers/Marlin#372] [Q&A] [answered]
Hello, got my v2 a week ago and now I have problems with bltouch wanted to try out this firmware now, but i dont know how to compile it for my bed since it is 230x230x250. Do anyone has an compiled version for my bed ?

**Comment by Jyers** [comment, marked as answer]
I'm assuming you have the default V2 bed, in which case despite what creality says, the actual bed size is 235x235. So 230x230 will not be an issue.

**Reply by ManuR93** [reply]
thx for your answer, so as i started abl, the first point was left of the bed in mid air, so i have to set x offset and after this Y offset for not hitting the bracket and then im fine ?

**Reply by AhJuicyKiwi** [reply]
I just flashed to this firmware after printing on creality firmware for the past month. But I have the same issue where Ill auto home and it'll knock Y past stop point and I don't hear a click then it'll home Y 2 more times then go to the middle. But it'll be way up in Z and when I do Auto Level or Home it'll move X,Y but Z only goes UP never touches the bed.. I went from being fine and dialed in to not even touching the bed or able to move Z down..... JYERS Please help my dude

**Reply by Jyers** [reply]
@Manu45er Yes, you need to set the x and y probe offsets in Control>Advanced to the distances between the probe and the nozzle in the z and y directions.

**Reply by Jyers** [reply]
@AhJuicyKiwi Sounds like either a hardware issue or a config mix up. Are you sure you are using the correct version for your board (ie. 4.2.2 vs 4.2.7)

**Reply by aryajuanda** [reply]
> thx for your answer, so as i started abl, the first point was left of the bed in mid air, so i have to set x offset and after this Y offset for not hitting the bracket and then im fine ?

yes, it's around -45 for X probe offset and -8 for Y Probe Offset, for more accurate value, you have to measure it by your self.
